pages/changelogs/send-your-feedback.mdx

```diff
@@ -0,0 +1,99 @@
+      description: "Engineer",
+      avatarUrl:
+        "https://june-changelog.s3.eu-central-1.amazonaws.com/alice_clavel_d41351cbdc.jpeg",
+    },
+    {
+      name: "Daniel Beere",
+      description: "Designer",
+      avatarUrl:
+        "https://june-changelog.s3.eu-central-1.amazonaws.com/T01_AS_9_R1_RDL_U03_C1_S7_F5_A5_90609690daa6_512_5071aad8fc.jpeg",
+    },
+    {
+      name: "Adis Banda",
+      description: "Engineer",
+      avatarUrl: "https://june-changelog.s3.eu-central-1.amazonaws.com/IMG_3227_939da5481a.PNG",
+    },
+    {
+      name: "Vinayak Mehta",
+      description: "Engineer",
+      avatarUrl:
+        "https://june-changelog.s3.eu-central-1.amazonaws.com/T01_AS_9_R1_RDL_U01_F830_FTC_7_94c9c8095b13_512_893705b288.jpeg",
+    },
+    {
+      name: "Enzo Avigo",
+      description: "Product",
+      avatarUrl:
+        "https://june-changelog.s3.eu-central-1.amazonaws.com/O_Doeqb_IW_400x400_47f624ed6e.jpg",
+    },
+  ],
+};
+
+Last week was the week after our 3.0 launch so we had to take some time to - onboard new users, fix any bugs that were reported during launch week and talk with many of you.
+
+After building June 3.0 over the past year we needed to take some time to regroup and figure out what we want to build next.
+
+Our designer [Daniel](https://twitter.com/DanielBeere) has been leading this work and as he had a lot of experience conducting user research from his time at CircleCI, he prepared our scripts and started interviewing users.
+
+In the spirit of sharing our learnings here are the questions we asked our users.
+
+## The context
+
+As we explore what we’ll do next to double down on B2B metrics and how we’ll make June more flexible.
+
+## Intro questions
+
+1. What are you using June for today? And what's the most valuable thing June does for you today?
+2. Could you tell me what made you decide to start using June? Did something happen in your company or team that inspired the decision?
+
+## Primary questions
+
+1. What other analytics tools do you use along with June? Could you tell me when you use June versus the other?
+2. What’s the biggest limit in June that restricts you from using it as your only analytics tool?
+3. Similar question, what’s the biggest limit that holds back the rest of your team from using June?
+4. Could you tell me a time when you used the information in one of your other analytics tools or June to take action or make a decision on something?
+5. Is there anything else you’d expect to get out of June?
+
+## Other tips on how to conduct user interviews
+
+1. Have a note-taking partner or use AI transcription
+2. Have a list of questions ready
+3. Involve the team in the interviews (get engineers to talk to customers)
+4. Be open minded, you're just there to listen. Don't mention or try to validate your own ideas.
+5. Don't always follow the script - get into follow up questions and let the conversation flow
+
+{/* Further reading suggestions */}
```

Comment:
1. Align with your team on your research goal and interview questions
2. Have a note-taking partner (or use AI transcription) 
3. Involve your team in the interviews (get engineers to talk to customers) 
4. Ask about the past instead of opinions about the future (ask interviewees to tell you a time when they experienced the problem and how they tried to solve it) 
5. Talk about their life instead of your ideas 

{/* Further reading suggestions */}
- https://www.momtestbook.com